cli/aws_orbit/remote_files/cdk/env.py

```diff
@@ -227,6 +227,7 @@ def _create_user_pool_client(self) -> cognito.UserPoolClient:
         return cognito.UserPoolClient(
             scope=self,
             id="user-pool-client",
+            access_token_validity=Duration.days(1),
```

Comment:
Hmm not sure why this fails the static checker, pulling this straight from: 
[https://docs.aws.amazon.com/cdk/api/latest/python/aws_cdk.aws_cognito/UserPoolClient.html#aws_cdk.aws_cognito.UserPoolClient](https://docs.aws.amazon.com/cdk/api/latest/python/aws_cdk.aws_cognito/UserPoolClient.html#aws_cdk.aws_cognito.UserPoolClient)

Reply:
I wonder how this would work when cognito is not the provider but just the broker, for example Okta is the provider.  Does this setup still works?  We should probably do some testing with the Okta setup as well.

Reply:
My understanding is that the UserPoolClient would work the same regardless of the idp provider, but I agree knowing the exact behaviors with both setups is critical.